### ISSUE TITLE
compile_attributes return error when error is not supposed to occur

### ISSUE THREAD

**Issue by earnaud** [EDIorg/EMLassemblyline#53]
Hi,

while using the `EMLassemblyline::make_eml()` function, an error occur which include `compile_attribute`. The error text is:
```
'The number of attributes listed in ',
          fname_table_attributes[i],
          ' does not match the number of columns listed in the corresponding data table. Please correct this. '
```
Actually, the described error is not met in the MetaShARK's scripts. A colleague had this issue happening with other files than the concerned ones, but even by correcting manually this, error still occurs.

By the way, you may try it yourself with the dev version of MetaShARK.

**Comment by clnsmth** [comment]
This error originates from a check between the column names in a data table and the attribute names in the attributes template. This check contains an inaccuracy that I should be able to fix soon.

Generally, what needs to be changed in EALs error and warning messages to improve integration with MetaShARK?

**Comment by clnsmth** [comment]
Related to https://github.com/EDIorg/EMLassemblyline/issues/35

**Comment by earnaud** [comment]
For general features required for EAL to be properly integrated into EAL, I would suggest moving on to a dedicated topic, like [this](https://github.com/earnaud/MetaShARK-v2/issues/26)

**Comment by clnsmth** [comment]
This has been fixed in f1c0f74afa7945f8c21fea7406a1d35a12dee345.
